linux_os/guide/services/deprecated/package_telnetd-ssl_removed/rule.yml

```diff
@@ -2,15 +2,23 @@ documentation_complete: true
 
 title: 'Uninstall the ssl compliant telnet server'
 
-description: 'The telnet daemon, even with ssl support, should be uninstalled.'
+description: |-
+        The <tt>telnet</tt> daemon, even with ssl support, should be uninstalled.
 
-rationale: 'telnet, even with ssl support, should not be installed. When remote shell is required, up-to-date ssh daemon can be used.'
+rationale: |-
+        <tt>telnet</tt>, even with ssl support, should not be installed.
+        When remote shell is required, up-to-date ssh daemon can be used.
 
 severity: high
 
+identifiers:
+    cce@rhel6: 82449-0
+    cce@rhel7: 82450-8
+    cce@rhel8: 82451-6
```

Comment:
@shawndwells please return these CCEs to the pool. There is no `telnetd-ssl` package.